management/management_test.go

```diff
@@ -13,26 +14,31 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var m *Management
-
 var (
-	domain       = os.Getenv("AUTH0_DOMAIN")
-	clientID     = os.Getenv("AUTH0_CLIENT_ID")
-	clientSecret = os.Getenv("AUTH0_CLIENT_SECRET")
-	debug        = os.Getenv("AUTH0_DEBUG")
+	domain          = os.Getenv("AUTH0_DOMAIN")
+	clientID        = os.Getenv("AUTH0_CLIENT_ID")
+	clientSecret    = os.Getenv("AUTH0_CLIENT_SECRET")
+	debug           = os.Getenv("AUTH0_DEBUG")
+	vcr             = os.Getenv("AUTH0_VCR")
```

Comment:
If we're referencing the VCR tests has HTTP recordings elsewhere, we should do the same here, and below.

Reply:
Thanks for spotting this! :) Updated! [2c3f9d8](https://github.com/auth0/go-auth0/pull/67/commits/2c3f9d8cf3d52dec455603ae5f549eae38e15f13)